repository/RediStick-Core.package/RsRediStick.class/instance/username.password..st

```diff
@@ -0,0 +1,4 @@
+setter
+username: aUsername password: aPassword 
+	self userName: aUsername.
```

Comment:
In this case, I prefer direct variable access, because this method is just an initializer.
```
userName := aUserName.
password := aPassword.
````

Reply:
fix 35150ae